notebooks/week5/04.deploy_model.py

```
@@ -0,0 +1,55 @@
+    action="store",
+    default=None,
+    type=str,
+    required=True,
+)
+
+args = parser.parse_args()
+root_path = args.root_path
+
+config_path = (f"{root_path}/project_config.yml")
+config = ProjectConfig.from_yaml(config_path=config_path)
+model_version = dbutils.jobs.taskValues.get(taskKey="evaluate_model", key="model_version")
```

Ruff

Undefined name `dbutils`

(F821)